### PR TITLE
Feat/update status texts for completions

### PR DESCRIPTION
**Pull request by kajm**
## Explain the changes you’ve made
Align texts on CaseOverview and CaseSummary screen with whats in Figma.
Also smaller logic refactoring and case status type cleanup.

## Explain why these changes are made
To make random check and completions works as expected, these changes are made.

## Explain your solution
Constant logic creation on both CaseOverview and CaseSummary are moved to a common function.
Completions are removed from CaseOverview screen and should only be visible in CaseSummary.
Removed unused case status types from type file
Make sure status types are used instead of dangling strings

## How to test

1. Checkout this branch
2. Fire upp the simulator by running the command `yarn ios` or `yarn android`
3. Start whichever form and see if it works as expected 

## Tested environments

- [] Storybook on a iOS device/simulator.
- [x] Building the Application on a iOS device/simulator.
- [] Building the Application on a Android device/simulator.

### CODE REVIEW

**Comment by helsingborgsstad** [comment]
Task linked: [CU-2qa1anj Uppdatera statustexter som berör completions](https://app.clickup.com/t/2qa1anj)

**Review by dannil76 — APPROVED** [review]

File: source/screens/FormCaseScreen.tsx [hunk 1]
```diff
@@ -120,37 +120,30 @@ const FormCaseScreen = ({
     currentPosition: FormPosition
   ): Promise<Action | void> => {
     // If the case is submitted, we should not actually update its data...
```
Comment (kajm) [inline source/screens/FormCaseScreen.tsx:122]: remove

Comment (kajm) [inline source/screens/FormCaseScreen.tsx:122]: remove

**Review by MasterKenth — APPROVED** [review]
untested

**Pull request merged**